tests/main/interfaces-openvswitch/task.yaml

```diff
@@ -15,8 +15,14 @@ details: |
     list and deletion.
 
 prepare: |
+    . "$TESTSLIB/pkgdb.sh"
+
     echo "Given openvswitch is installed"
-    apt install -y --no-install-recommends openvswitch-switch
```

Comment:
Fixed that. `distro_install_package` now accepts `--no-install-recommends` as argument.